datasetinsights/io/tracker/factory.py

```diff
@@ -0,0 +1,19 @@
+from datasetinsights.constants import MLFLOW_TRACKER
+from datasetinsights.io.tracker.mzflow import MLFlowTracker
+
+
+class TrackerFactory:
+    """Factory: responsible for creating instance of tracker class"""
+
+    @staticmethod
+    def create(config, tracker_type):
+        """create tracker class object.
+
+        Args:
+            config : config object, holds server details
+            tracker_type : type of tracker
+        """
+        if MLFLOW_TRACKER == tracker_type:
+            return MLFlowTracker.get_tracker(config)
```

Comment:
remember to change this to call the CTOR with the client_id etc. so the factory keep the reference to the singleton rather than the tracker itself doing that